identity-core/src/crypto/merkle_key/tag.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/// A tag identifying a Merkle Key Collection signature or digest algorithm.
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]
+#[repr(transparent)]
+pub struct MerkleTag(u8);
```

Comment:
May add to it in the future but any implementation would need to be added here anyways

There are also already `MerkleDigest` and `MerkleSignature` traits which use this type. not sure what adding another trait for this would do?

Reply:
> any particular reason why the feature flags for those 3 from crypto.rs (i.e. "blake2b") are not exposed as flags in the identity crates

This would be awesome to have but hasn't been a priority. it's a bit involving since you need to coordinate features across all crates.

Some things like `blake2b` are also fundamental to the method spec and shouldn't be disabled anyways